cbt_enable_and_snapshot.py

```diff
@@ -20,6 +20,14 @@
 import argparse
 
 
+def enable_NBD(session):
+    # Not sure if we want to make nbd/nbd_insecure an option here

```

Comment:
This is the only question I have for this commit

Reply:
I don't think we need this, as this example is showing the better (secure) variant. Would suggest to remove this comment.